### DISCUSSION TITLE
Status line layout in the README

### DISCUSSION THREAD

**Discussion by bouteillerAlan** [catppuccin/nvim#359] [Q&A] [answered]
Hello :wave:  

Is the config file allowing to have the same layout at the screenshot in the readme is available somewhere ?

**Comment by rewhile** [comment, marked as answer]
It's our default feline integration (not lualine), here is my config:

```lua
use {
	"feline-nvim/feline.nvim",
	after = "catppuccin",
	config = function()
		require("feline").setup {
			components = require("catppuccin.groups.integrations.feline").get(),
		}
	end,
}
```

**Reply by bouteillerAlan** [reply]
Ho ok ! Thanks :+1: 

Maybe a hint about that in the readme can be useful?

**Reply by jemorriso** [reply]
FYI to anyone who sees this, feline.nvim has been deprecated but there is a fork, 'freddiehaddad/feline.nvim', that I am able to use with the above config and it works fine.

**Comment by Tokito-to** [comment]
feline.nvim is archived since 2023 any other alternatives with good integration for catppuccin

**Reply by JAWS-tm** [reply]
You can use [lualine.nvim](https://github.com/nvim-lualine/lualine.nvim?tab=readme-ov-file) with the [bubbles](https://github.com/nvim-lualine/lualine.nvim/blob/master/examples/bubbles.lua) theme. But note that the colors won’t be the same as catppuccin theme from `feline.nvim`.
